server/service/src/invoice/inventory_adjustment/adjust_existing_stock/insert.rs

```diff
@@ -307,12 +348,12 @@ mod test {
 
         assert_eq!(
             updated_stockline.available_number_of_packs,
-            mock_stock_line_a().available_number_of_packs + 3.0
+            mock_stock_line_a().available_number_of_packs + 2.0
```

Comment:
We had two adjustments in one test (+1 and +2), which made this comparison confusing to me. So separated them 🤷‍♀️